### ISSUE TITLE
bookables need a location

### ISSUE THREAD

**Issue by monkeypants** [gs-gs/parks-bce-integration-bookable-things#6]
We will want bookable things on a map. So the bookable things need a location.

Some things will need a point - like a BBQ or shop or campsite.

Some things will need an area or a line, like a tour.

Some things will need word descriptions too "14 paces past the yellow stump, in the direction of the full moon on the first night of summer". GIS is better.

if present, coordinates should be returned by the GET bookable thing API.

In the future, one bookable thing may have a collection of geospatial features. Meet the ranger HERE, and get picked up over THERE two days later. On the way, we will be passing through THESE landmarks (all points) following this track (line), and exploring this area (polygon). We will be staying out of these areas (polygons) because they are closed to us.

**Comment by monkeypants** [comment]
when an area is closed...

**Comment by koriaf** [comment]
we could attach a KMZ file for patch, and just save a set of GPS coordinates with short description ("XX:XX - park here", "YY:YY - meeting point") to the bookable thing itself (assuming it won't change between different dates - it's like they start at different places on different days of week).

Word description could fit existing "short_description" text field or a new one could be created for it.

Regarding the closed areas: they just could disable some slots so it's impossible to place a reservation for these days

**Comment by monkeypants** [comment]
Yes, a kml file (or geojson) could be saved as an attribute of the bookable thing. Or we could have a full PostGIS layer of "landmarks", and a M:N relationship between bookable thing and landmarks.

**Comment by monkeypants** [comment]
Reasons to favor the "layer of landmarks" design:
* landmarks could be shared; this tour meets at this cafe, which is also where that your meets.
* there are likely to be "what else can I do here" type queries/use-cases
* we probably want a curated landmark database (chose landmarks from the map, request to have landmarks added to the map, etc)

**Comment by monkeypants** [comment]
(of course we could have both - landmarks in a DB and then generate the geojson/klm from it. But referential integrity might be more of a pain when the map changes)

**Comment by monkeypants** [comment]
(FYI @adamkerz; this is the ticket that relates to our discussion about GIS and bookable things...)
